viz_scripts/generic_metrics.ipynb

```diff
@@ -215,19 +216,33 @@
    },
    "outputs": [],
    "source": [
-    "plot_title_no_quality=\"Mode confirmations for trips under 10 Miles\"\n",
+    "#determine 80th percentile\n",
+    "cutoff = expanded_ct.distance.quantile(0.8)\n",
+    "\n",
+    "if use_imperial:\n",
+    "    dist_unit = \"miles\"\n",
+    "    dist_threshold = expanded_ct.distance_miles.quantile(0.8).round(1)\n",
```

Comment:
here, it is different, if using imperial distance then the unit string in the labels is "miles" and the cutoff distance `dist_threshold` is conveyed in miles, for metric units both of those things are in kilometers

`cutoff` above is the raw distance (meters) that is what actually filters the data, the if/else just handles the two cases of labels

Reply:
I have now streamlined / deduplicated the if/else statements, using string variables as much as possible. There are still some that are similar, but switching them with a variable would be just as much code ... and typing that out I'm not sure that's a bad thing, if it meant all of the variables were assigned at once, it would be clear where things were getting switched ... I'm going to make one more pass